content/_partials/books.adoc

```diff
@@ -0,0 +1,3 @@
+NOTE: The list of books on this page is provided for informational purposes.
+The books are not endorsed by Jenkins project.
+Older books on the list contain some outdated information.
```

Comment:
Maybe we should disclose the order is by date, in case this is not obvious to the reader.